lib/src/base-styles/_nx-lists.scss

```diff
@@ -128,109 +120,65 @@
  * Action or Clickable Lists
  */
 .nx-list--clickable {
-  .nx-list__title {
-    color: $nx-text-color;
-  }
-
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
-      pointer-events: none;
-    }
+      background-color: #d3e6ff;
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
-      &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
-        cursor: pointer;
-        outline-style: none; // removes :focus outline
-
-        .fa-chevron-right {
-          color: $nx-color-blue-41;
-        }
-      }
-
-      &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+      &:hover {
+        background-color: #d3e6ff;
       }
     }
 
-    &.nx-list__item--empty {
-      @include regular();
-
-      border-top: $nx-border; // assume it's always the first item
-      color: $nx-text-color-light;
+    &:hover {
+      background-color: #f1f3fa;
+      cursor: pointer;
+      outline-style: none; // removes :focus outline
```

Comment:
This makes me realize that clickable list items should probably 
1. be focusable
2. have a focus style

Reply:
> This makes me realize that clickable list items should probably
> 
> 1. be focusable
> 2. have a focus style

Focus added here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/a662ee27667f2b916ef6a42ec0fa046cd4eb6603 and tweaked here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/b00e86b24b8caf9ddcb4ce73f56d01e4192b9d5f